components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.html

```diff
@@ -18,10 +18,14 @@ <h2>Add Member Expression</h2>
           <form [formGroup]="expressionForm">
             <chef-form-field>
               <label>Member expression *</label>
-              <input chefInput formControlName="expression" type="text">
+              <input chefInput formControlName="expression" type="text" (keyup)="resetErrors()" >
```

Comment:
As soon as the value is edited, clear the error so it can then appear again if it needs to later.